src/Components/Main/TodoForRender/TodoForRender.tsx

```diff
@@ -0,0 +1,35 @@
+import cn from 'classnames';
+import { useTodoContext } from '../../../Context/Context';
+import { RemoveButton } from '../RemoveButton/RemoveButton';
+import { TodoLoader } from '../TodoLoader/TodoLoader';
+
+export const TodoForRender = () => {
+  const { filteredList } = useTodoContext();
+
+  return (
+    <>
```

Comment:
```suggestion
    <section className="todoapp__main" data-cy="TodoList">
```
  
 `TodoMain` component is redundant as it does nothing, just render the `TodoList`